cache_images/fedora_packaging.sh

```diff
@@ -177,11 +178,7 @@ fi
 # Download these package files, but don't install them; Any tests
 # wishing to, may install them using their native tools at runtime.
 DOWNLOAD_PACKAGES=(\
-    oci-umount
     parallel
-    podman-docker
```

Comment:
Is this not a thing anymore?  I (very) vaguely remember this being referenced somewhere less-traveled (like Buildah CI or gitlab CI).

Reply:
sure it exists, we can install this from the copr / distro repos itself. Unless I'm reading that wrong.

Reply:
Maybe the git logs will help find where this was added...

Reply:
...found it!  af40a752ace775239a71f5f62082584008d745da

Reply:
the skopeo tests are failing so i added plugins and podman-docker back.